packages/contentstack-branches/src/utils/create-merge-scripts.ts

```diff
@@ -7,8 +7,9 @@ import { assetFolderCreateScript } from './asset-folder-create-script';
 
 type CreateMergeScriptsProps = {
   uid: string;
-  entry_merge_strategy: string;
+  entry_merge_strategy?: string;
   type?: string;
+  status: string
```

Comment:
@cs-raj , can you check whether it's optional or mandatory otherwise it will break script generation? Formatting missing

Reply:
Done